addons/l10n_co_pos_details/wizard/pos_details.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class PosDetails(models.TransientModel):
+    _name = 'l10n_co_pos.details.wizard'
+
+    def _default_start_date(self):
+        """ Find the earliest start_date of the latests sessions """
+        # restrict to configs available to the user
+        config_ids = self.env['pos.config'].search([]).ids
+        # exclude configs has not been opened for 2 days
+        self.env.cr.execute("""
+            SELECT
+            max(start_at) as start,
+            config_id
+            FROM pos_session
+            WHERE config_id = ANY(%s)
+            AND start_at > (NOW() - INTERVAL '2 DAYS')
```

Comment:
@nla-odoo Could you please use a regular search/read_group instead of this ugly SQL?